data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -167,40 +193,37 @@ def _climate_factor(self) -> bool:
             )
         )
 
-        return (
-            self.df[
-                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
-                + field_names.PERCENTILE_FIELD_SUFFIX
-            ]
-            >= self.LOW_INCOME_THRESHOLD
-        ) & climate_criteria
+        return self.FPL_200_SERIES & climate_criteria
 
     def _energy_factor(self) -> bool:
         # In Xth percentile or above for DOE’s energy cost burden score (Source: LEAD Score)
         # AND
         # Low income: In 60th percentile or above for percent of block group population
         # of households where household income is less than or equal to twice the federal
-        # poverty level. Source: Census's American Community Survey]
-        energy_criteria = (
+        # poverty level. Source: Census's American Community Survey]  
+
+        threshold_one = (
```

Comment:
Nit: can we use a more meaningful name here? Like `energy_burden_threshold`? Just helps to reduce future errors.